src/utils/time.ts

```diff
@@ -53,57 +86,23 @@ export const secondsToDuration = (totalSeconds: number): Duration => {
 export const formatSecondsAsDuration = (seconds: number): string =>
   formatDuration(secondsToDuration(seconds));
 
-const formatMeasurementInString = (value: number, measureUnit: string) => {
-  const baseFormatted = `${value} ${measureUnit}`;
-
-  if (value === 0) return "";
-  if (value === 1) return baseFormatted;
-  if (value > 1) return `${baseFormatted}s`;
-};
-
-// @debt quality test this
 export const getTimeBeforeParty = (startUtcSeconds?: number) => {
```

Comment:
[NB] Does this need to allow `undefined` params?

Reply:
[NB] It would be good to set an explicit return type on this function too.

Reply:
At first, I removed that `?` to disallow `undefined` but then the app broke and I decided to keep it as is, didn't dig deep into it. :see_no_evil: 

added JSDoc and return type to the function: https://github.com/sparkletown/sparkle/pull/1426/commits/64e2b9d2c1f4b2cf715ff86330b85947d012a966